packages/shared/src/hooks/bookmark/useBookmarkFolderList.ts

```diff
@@ -0,0 +1,21 @@
+import { useQuery } from '@tanstack/react-query';
+import { BookmarkFolder, getBookmarkFolders } from '../../graphql/bookmarks';
+import { generateQueryKey, RequestKey, StaleTime } from '../../lib/query';
+
+interface UseBookmarkFolderList {
+  isPending: boolean;
+  folders: Array<BookmarkFolder>;
+}
+
+export const useBookmarkFolderList = (): UseBookmarkFolderList => {
+  const { data, isPending } = useQuery({
```

Comment:
Can probably be disabled if no user found right, as it sits in sidebar?

Reply:
Updated ✅